### PR TITLE
Fall Banquet Info

### PR DESCRIPTION
**Pull request by amicableskeptic**
Please merge and deploy if all looks good.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -134,7 +134,7 @@ <h2 class="title-margin-bottom">News &amp; Events</h2>
...
-                   <p>For those who missed a chance to reunite with old friends and meet new, you are invited to our Fall Banquet Honoring the 1967 Women’s Championship Team, October 14th 2017 in New York City.</p>
+                   <p>For those who missed a chance to reunite with old friends and meet new, you are invited to our Fall Banquet Honoring the 1967 Women’s Championship Team, September 23, 2017 at the Cornell Club in New York City. <a href="http://cornelluniversity.imodules.com/s/1717/alumni/event.aspx?sid=1717&gid=2&pgid=9077&content_id=9447" target="_blank">Register here</a></p>
```
Comment (NickHeiner) [inline app/index.html]: Let's put a period at the end of the sentence "Register here".

Comment (NickHeiner) [reply app/index.html]: Just for fun, would you mind adding GA click tracking to these links? If that'll be too cumbersome, let me know, and I'll do it if I want it that badly. :smile:

**Review by NickHeiner — CHANGES_REQUESTED** [review]
Just one small change, then we can deploy.

File: app/index.html (hunk 1 above)
Comment (amicableskeptic) [reply app/index.html]: I'm confused by this request. I looked at all the other links on the page and I don't see anything having been done to them to add GA click tracking. I then read Google's Set up Analytics tracking instructions (https://support.google.com/analytics/answer/1008080?hl=en) and they only talk about pasting the entire Javascript tracking-code snippet into your web page and not doing stuff to particular links. I checked and it looks like the GA snippet is in there. Am I missing something?

**Comment by NickHeiner** [comment]
Also, I recommend that you make a feature branch and send PRs from that going forward, and keep your fork's `dev` as a mirror of the upstream `dev`. That should make your life easier.

File: app/index.html (hunk 1 above)
Comment (NickHeiner) [reply app/index.html]: I see the GA network requests when the link is clicked, so I assume that we are tracking this. Sorry for the confusion.

**Review by NickHeiner — APPROVED** [review]

**Comment by NickHeiner** [comment]
Deployed!

**Pull request merged**